src/components/VideoCards/VideoCards.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import VideoCards from './index';
+
+test('renders all videos of mockfile', () => {
+    render(<VideoCards />);
+
+    expect(screen.getByText(/Video Tour/i)).toBeInTheDocument();
```

Comment:
I don't think it is needed to test that every card gets rendered. It would be better to test that the size of the expected cards match to the ones that you provided, and instead test the card separately in its own test file. To check the title, description and so on.